.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: "1.18"
+          go-version: "1.20"
```

Comment:
The version has been modified in the tests because the tests check the documentation generation by comparing raw strings, and in 1.19 there has been changes to `go fmt` regarding doc comments, so that makes tests harder to check if they aren't on the latest `go fmt` version